pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -239,6 +242,9 @@ const (
 	Topics = `topics`
 )
 
+// Support additional mechanism on top of the default SASL mechanism.
+const SASLTypeAwsIAMRole = "AWS_MSK_IAM"
```

Comment:
nit: change the name of the constant too pls

Reply:
Nice catch, I forgot that, btw I don't fully understand the casing convention in the names, because Aws is written with capitalized A, and IAM, SASL(and MSK I guess) are written as all caps. Meanwhile, GCP is in all caps, not sure if `SASLTypeAwsMSKIAM` is complaint to the CRL naming convention.

Reply:
@cty123 I think the most correct name would be `SASLTypeAWSMSKIAM` but that's kind of hard to read. but now that you mention it, why dont you go ahead and change it to that ;)

Reply:
Are you sure about that, because I see in other place AWS is always written as `Aws` like here: https://github.com/cockroachdb/cockroach/blob/46a5cb142d7f9c19c190d63dc7c8f91195e4ef3c/pkg/roachprod/vm/aws/aws.go#L59

Reply:
I changed to `SASLTypeAWSMSKIAM` anyways, I see both casing being used in code